src/apps/content-editor/src/app/components/ReorderNav/ReorderNav.less

```diff
@@ -2,6 +2,8 @@
 @import "~@zesty-io/core/typography.less";
 
 .ReorderNav {
+  overflow-y: auto;
+  height: 70vh;
```

Comment:
this may be better served as a `max-height` so short lists do not use a large modal but long lists are bounded